var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -162,6 +162,9 @@ class Cmake(Package):
     depends_on("gmake", when="platform=freebsd")
 
     depends_on("qt", when="+qtgui")
+    for plat in ["linux", "darwin", "freebsd"]:
+        with when(f"platform={plat}"):
+            depends_on("qt ^libmng build_system=autotools", when="+qtgui")
```

Comment:
Our audit forbids `depends_on(x ^y)`, it wants you to do

```
depends_on(x)
depends_on(y)
```

Instead

Reply:
Thanks Peter!

Reply:
Doesn't that force a direct dependency on `y` when one doesn't exist for the package? 
I made the change, just curious about modeling.

Reply:
I think the correct thing is to label it is a `runtime` dependency.